content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -127,14 +127,15 @@ instructions.
 
 ### CSI driver restrictions
 
-{{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
+CSI ephemeral volumes allow users to provide `volumeAttributes`
+directly to the CSI driver as part of the Pod spec. A CSI driver
+requiring `volumeAttributes` that are typically restricted to
+administrators is NOT suitable for use in an inline ephemeral volume.
 
-As a cluster administrator, you can use a [PodSecurityPolicy](/docs/concepts/security/pod-security-policy/) to control which CSI drivers can be used in a Pod, specified with the
-[`allowedCSIDrivers` field](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podsecuritypolicyspec-v1beta1-policy).
-
-{{< note >}}
-PodSecurityPolicy is deprecated and will be removed in the Kubernetes v1.25 release.
-{{< /note >}}
+Cluster administrators who need to restrict the CSI drivers that are
+allowed to be used as inline volumes within a Pod spec may do so by:
+- Removing `Ephemeral` from `volumeLifecycleModes` in the CSIDriver spec.
```

Comment:
Removing `Ephemeral` from `volumeLifecycleModes` in the CSIDriver spec means this CSI Driver cannot support CSI ephemeral volumes, right?  Can you clarify?

Reply:
That's correct, I updated this line to clarify.